src/generic/datavgen.cpp

```diff
@@ -2468,6 +2468,9 @@ wxBitmap wxDataViewMainWindow::CreateItemBitmap( unsigned int row, int &indent )
     width -= indent;
 
     wxBitmap bitmap( width, height );
+#ifdef __WXMSW__
+    bitmap.SetScaleFactor( GetDPIScaleFactor() );
+#endif // __WXMSW__
```

Comment:
Could you please check that the following code, which should be equivalent, but avoids platform-specific code, work too?

```suggestion
    wxBitmap bitmap;
    bitmap.CreateWithLogicalSize( width, height, GetDPIScaleFactor() );
```